docs/ide/tutorial-csharp-aspnet-core.md

```diff
@@ -192,4 +192,4 @@ Congratulations on completing this tutorial! We hope you learned a little bit ab
 
 ## See also
 
```

Comment:
@jmayntzhusen I am hesitant about making this change because the link works as is, but we can try it and see if the change passes our build verification process.